scss/_variables.scss

```diff
@@ -393,4 +388,46 @@ $icon-spacing: 4px !default;
 
 // Disabled mixin variables
 $disabled-filter: grayscale(.1) !default;
-$disabled-opacity: .6 !default;
\ No newline at end of file
+$disabled-opacity: .6 !default;
+
+// Window
+$window-inner-padding-x: 16px !default;
+$window-inner-padding-y: 16px !default;
+$window-titlebar-padding-left: 16px !default;
+$window-titlebar-padding-right: 16px !default;
+$window-titlebar-padding-top: 16px !default;
+$window-titlebar-padding-bottom: 16px !default;
+
+$window-titlebar-background-image: none !default;
+$window-titlebar-background-color: $accent !default;
+$window-titlebar-color: $accent-contrast !default;
```

Comment:
This is a nitpick, but `background` is shortened to `bg` in all variable names, and text color uses the suffix `-text` rather than `-color`.

Also, what do you think about using only one variable that sets the shorthand value? This will allow users to set the `background-repeat` and `background-position`, too.